tokio/src/runtime/builder.rs

```diff
@@ -677,6 +690,91 @@ impl Builder {
         self
     }
 
+    /// Executes function `f` just before a task is spawned.
```

Comment:
Does it make sense to add a description explaining what will happen when the provieded f gets paniced?

Reply:
I didn't think that would add value. Would you like it present?

Reply:
Feel free to take it or leave it, I don’t think it’s something that would block the merge!

Reply:
I'll leave it out for now, on the assumption that this will be getting documentation polish later on anyways.